src/menu.js

```diff
@@ -0,0 +1,216 @@
+export default function createMenu(content) {
```

Comment:
- [ ] You should be using `ES6` syntax in this project so here you should use an `arrow function` instead of the `function` keyword. 
Screenshot of requirements 👇🏾 
<img width="470" alt="Screen Shot 2021-04-15 at 8 59 37 PM" src="https://user-images.githubusercontent.com/48804258/114923765-7e9f0600-9e2d-11eb-9cdb-a70153518a83.png">